### PR TITLE
Minor fixes and improvements

### PR DESCRIPTION
**Pull request by click0**

### CODE REVIEW

File: depenguinme.sh [hunk 1]
```diff
@@ -164,6 +164,8 @@ done
 
 if [ "$REQUIRE_SSHKEY" = "YES" ]; then
 	[ -s COPYKEY.pub ] || exit_error "Authorized key sources are empty"
+else
```
Comment (grembo) [inline depenguinme.sh:169]: The command in line 155 already creates an empty file, so this isn’t necessary

File: depenguinme.sh [hunk 2]
```diff
@@ -58,7 +58,7 @@ usage() {
 }
 
 is_url() {
-	[[ "$1" =~ ^(http|https|ftp):// ]]
+	[[ "$1" =~ ^(http|https|ftp|sftp):// ]]
```
Comment (grembo) [inline depenguinme.sh:61]: Is sftp supported by wget?

**Review by grembo — CHANGES_REQUESTED** [review]

File: depenguinme.sh (hunk 1 above)
Comment (grembo) [reply depenguinme.sh:169]: Maybe go explain, “:” means “true” and >filename overwrites filename, so `: >COPYFILE.pub` makes sure an empty file of that name exists (regardless if it existed beforehand or not)

File: depenguinme.sh (hunk 2 above)
Comment (bretton) [reply depenguinme.sh:61]: "unsupported scheme 'sftp'" on my test box

**Comment by bretton** [comment]
build test fine. login good.

File: depenguinme.sh (hunk 2 above)
Comment (click0) [reply depenguinme.sh:61]: Yes, not supported :(

File: depenguinme.sh [hunk 3]
```diff
@@ -113,6 +113,8 @@ apt-get install -y qemu
 QEMUBASE="/tmp/depenguinme"
 USENVME=0
 MYPRIMARYIP=$(ip route get 1 | awk '{print $(NF-2);exit}')
+MYSSHEXTERNALPORT="1022"
```
Comment (grembo) [inline depenguinme.sh:116]: Should this be somehow overrideable (either environment or preferably command line option)?

**Review by grembo — CHANGES_REQUESTED** [review]

**Comment by grembo** [comment]
@click0 I think there could be some value in making portforwarding configurable, maybe a separate PR to just implement this would make sense. If you require the feature, but you’re not 100% how to implement, I could create a PR myself (but only if you see an actual need).

**Comment by grembo** [comment]
@bretton even though the pull request has been closed, maybe adding .idea might be a *drummroll* good idea, as it is a file/directory created by IntelliJ (I had - even bigger *drummroll* - no idea it was). ;)

**Pull request closed without merging**